frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/mock-context.tsx

```diff
@@ -12,16 +13,32 @@ import {
 import { connect } from "metabase/lib/redux";
 
 export type MockDashboardContextProps = Partial<
-  PropsWithChildren<DashboardContextReturned>
+  Omit<PropsWithChildren<DashboardContextReturned>, "dashboardActions"> & {
+    dashboardActions: DashboardContextOwnProps["dashboardActions"];
+  }
 >;
 // Create a component that accepts all redux props and passes them into DashboardContext
 const DashboardContextWithReduxProps = (
   props: PropsWithChildren<DashboardContextReturned>,
-) => (
-  <DashboardContext.Provider value={props}>
-    {props.children}
-  </DashboardContext.Provider>
-);
+) => {
+  const {
+    isEditing,
+    downloadsEnabled,
+    dashboardActions: dashboardActionsOrGetter,
+  } = props;
+
+  // Use exact same implementation as in DashboardContextProviderInner
```

Comment:
This fixes the [failed unit tests](https://github.com/metabase/metabase/actions/runs/16023766646/job/45206662124?pr=60198). It happens because we lift the `dashboardActions` up to the dashboard context, and we need to provide this value to the tests that are missing it. Additionally, we accept a dashboardAction getter too, so we need to replicate the same functionality here.

https://github.com/metabase/metabase/blob/cf1985c6b4d00d4c2fedd87939c1e34a1a2a8c45/frontend/src/metabase/dashboard/context/context.tsx#L357-L360